Remotion/Web/Development.WebTesting/WebTestConfigurationFactory.cs

```diff
@@ -122,7 +122,7 @@ public IHostingConfiguration CreateHostingConfiguration ()
     }
 
     /// <summary>
-    /// Responsible for handling Browsers other than Chrome and Internet Explorer.
+    /// Responsible for handling Browsers other than Chrome, Edge, and Firefox.
```

Comment:
```suggestion
    /// Responsible for handling Browsers other than Chrome, Edge and Firefox.
```

Reply:
Nope, starting with three options, English requires a comma before the `and`

Reply:
I didn't know the oxford comma is now mandatory in the english language. imo it just looks weird. Anyway, if it's required 🤷🏼‍♂️

Reply:
Interesting, didn't know it was only for them fancy talkers required. Anyhow, I think for me, it dates back to the prep lessons for my own masters thesis and I don't want to dig up whether it's actually required or just good practice for formal text. Either way, it's official enough to keep it when it's there, but may be okay to not mention it when it's missing during a review.